<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>simple-gui-downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>611</width>
      <height>450</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="line">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Paste a link (YouTube, Bandcamp and etc)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="outpath_line">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outpath_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Choose Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QPushButton" name="button">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Download</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="quality_box">
             <property name="currentText">
              <string>Best</string>
             </property>
             <item>
              <property name="text">
               <string>Best</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>360</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>480</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>720</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1080</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1440</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Qality:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="video_button">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Video</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="audio_button">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Audio</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QCheckBox" name="thumbnail_checkbox">
             <property name="text">
              <string>Thumbnails</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="playlist_checkbox">
             <property name="text">
              <string>Full playlist</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="youtube_cover_checkbox">
             <property name="text">
              <string>Album cover (YouTube)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output folder:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="video_format_box">
           <item>
            <property name="text">
             <string>mp4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mkv</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>webm</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="audio_format_box">
           <item>
            <property name="text">
             <string>mp3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>opus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m4a</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>flac</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>aac</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>alac</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ffmpeg + yt-dlp folder:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="libpath_line">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="libpath_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Choose Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Settings:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="command_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
